single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java

```diff
@@ -59,7 +63,8 @@ public ServerConfiguration(
             long connectionHealthCheckPeriodInSeconds,
             long healthReadTimeoutInMilliseconds,
             int failureResponseCountThreshold,
-            Consumer<HttpRequest.Builder> requestTransformer
+            Consumer<HttpRequest.Builder> requestTransformer,
+            String protocol
```

Comment:
can you add a new constructor without protocol to avoid breaking changes and revert the change in the tests? Then add another test with http protocol?

Reply:
ah yes, good one, done